draft-wing-cidfi.md

```diff
@@ -703,12 +703,31 @@ update more frequently than once every second.
 
 The metadata exchanged over this channel is described in {{metadata-exchanged}}.
 
-## Ongoing Metadata Exchange {#ongoing-metadata-exchange}
 
-For the duration of the primary QUIC connection between the client and
-server, the client relays CNEs metadata changes to the server, and server's
-transmitted QUIC Destination CID to the CNEs.
+# Ongoing Signaling {#ongoing}
 
+Due to environmental changes on wireless networks or other user's
+traffic patterns, a particular flow may be able to operate faster or
+might need to operate slower.  The affected CIDFI Network Element SHOULD signal
+such conditions to the client ({{network-to-host}}), which can then
+relay that information to the server using either CIDFI or via its
+application.
+
+For example, a client streaming video might be retrieving low quality
+video because one of their CIDFI network elements indicated constrained
+bandwidth.  Later, after moving closer to an antenna, more bandwidth is
+available which is signaled by the CIDFI network element to the client.
+The client uses that signal to now request higher-quality video from
+the server.
+
+Similarly, the CIDFI client may begin receiving traffic
+with different characteristics which SHOULD be signaled to the CIDFI
```

Comment:
```suggestion
with different characteristics which might be be signaled to the CIDFI
```

Reply:
This is actually up to the application client whether to ask for differentiated behaviors for these streams.